src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,42 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int DEFAULT_START_VALUE = -1;
+    private static final int NUMBERS_CAPACITY = 12;
+
+    private int[] numbers = new int[NUMBERS_CAPACITY];
+    private int total = DEFAULT_START_VALUE;
+
+    public int getTotal() {
+        return total;
+    }
+
+    public void countIn(int in) {
+        if (!isFull())
+            numbers[++total] = in;
+    }
+
+    public boolean callCheck() {
+        return total == DEFAULT_START_VALUE;
+    }
+
+    public boolean isFull() {
+        return total == 11;
```

Comment:
This constant is related to NUMBERS_CAPACITY . Changing only one leads to an error (inconsistency).